### DISCUSSION TITLE
Resolve RBAC group names

### DISCUSSION THREAD

**Discussion by pauleboy** [Azure/CCOInsights#264] [Q&A]
Hi Team

Is there a way for the dashboard to resolve AzureAD group names instead of just by id I cannot see it within the tables? Also is there a way to query to expand the AD group to resolve group membership?

Many thanks

**Comment by josunefon** [comment]
Hi @pauleboy 

The names of the groups are appearing:
![image](https://user-images.githubusercontent.com/39730064/204361637-72371327-438c-4d93-a31d-ca5ba2545ac5.png)

In some cases with autogenerated groups/users the mailnickname or display name it's a GUID but in all the other cases it should be tha "normal name"

Can you share an image of your problem to see it?
